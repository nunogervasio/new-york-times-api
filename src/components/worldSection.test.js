import React from "react";
import { shallow } from "enzyme";
import WorldSection from "./WorldSection";

describe("WorldSection", () => {
  const worldSection = shallow(<WorldSection />);

  it("renders properly", () => {
    expect(worldSection).toMatchSnapshot();
  });
  it("renders SectionCard", () => {});
  it("renders PaginationComponent", () => {});
  it("handlePageChange", () => {});
});
