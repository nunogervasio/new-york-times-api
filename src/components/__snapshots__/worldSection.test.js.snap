// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WorldSection renders properly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WorldSection />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "centered": true,
      "children": <GridColumn
        computer={10}
        largeScreen={8}
        mobile={14}
        tablet={12}
        widescreen={6}
      >
        <PaginationComponent
          currentPage={1}
          itemsCount={0}
          onPageChange={[Function]}
          pageSize={4}
        />
      </GridColumn>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          Array [],
          <PaginationComponent
            currentPage={1}
            itemsCount={0}
            onPageChange={[Function]}
            pageSize={4}
          />,
        ],
        "computer": 10,
        "largeScreen": 8,
        "mobile": 14,
        "tablet": 12,
        "widescreen": 6,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "currentPage": 1,
            "itemsCount": 0,
            "onPageChange": [Function],
            "pageSize": 4,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "centered": true,
        "children": <GridColumn
          computer={10}
          largeScreen={8}
          mobile={14}
          tablet={12}
          widescreen={6}
        >
          <PaginationComponent
            currentPage={1}
            itemsCount={0}
            onPageChange={[Function]}
            pageSize={4}
          />
        </GridColumn>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": Array [
            Array [],
            <PaginationComponent
              currentPage={1}
              itemsCount={0}
              onPageChange={[Function]}
              pageSize={4}
            />,
          ],
          "computer": 10,
          "largeScreen": 8,
          "mobile": 14,
          "tablet": 12,
          "widescreen": 6,
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "currentPage": 1,
              "itemsCount": 0,
              "onPageChange": [Function],
              "pageSize": 4,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": true,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__childContext__): null,
}
`;
